### PR TITLE
[SP-5255] Create Incident refinements

### PR DESCRIPTION
**Pull request by moshebe**

### CODE REVIEW

**Review by hagaishapira — CHANGES_REQUESTED** [review]
Like we've discussed we need full parameterization of all items in the context.

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml [hunk 1]
```diff
@@ -47,7 +48,7 @@ steps:
     env:
       CHANNEL_NAME: "{{ $.event.ID }} {{ $.event.Name }}"
       PURPOSE: '{{ $.event.Description }}'
-      TOPIC: "StackPulse War Room Channel for Incident {{ $.event.Name }} ({{ $.event.ID }})\n Zoom meeting link - {{ $.join_url }}"
+      TOPIC: "StackPulse War Room Channel for Incident {{ $.event.Name }} ({{ $.event.ID }})\n {{- if $.join_url }} Zoom meeting link - {{ $.join_url }} {{ end }}"
```
[the remark this answers is not in the document]
Comment (moshebe) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml:51]: It removes the whitespaces before the go-template expression

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml [hunk 2]
```diff
@@ -5,16 +5,39 @@ metadata:
 parameters:
   - name: notificationChannel
     type: var
-    description: "The name of the channel you would like to notify about the incident (without a #)."
+    description: 'The name of the channel you would like to notify about the incident (without a #).'
   - name: invitees
     type: var
-    description: "The emails of the users that will be invited to the channel (by default, only the incident reporter)."
+    description: The emails of the users that will be invited to the channel (by default, only the incident reporter).
```
Comment (hagaishapira) [inline stackpulse/create-incident-war-room-opsgenie/playbook.yaml:11]: let's be consistent with double, single or no quotes.

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml [hunk 3]
```diff
@@ -5,16 +5,39 @@ metadata:
...
     default: '{{ $.event.Reporter }}'
   - name: zoomIntegrationName
     type: integration
     description: Zoom integration.
     options:
-      integration_type: zoom
+      integration_type: zoom  
+  - name: eventID
```
Comment (hagaishapira) [inline stackpulse/create-incident-war-room-opsgenie/playbook.yaml:18]: This is the incident ID. please think about this playbook like being run by someone who isn't very familiar with the platform, provide good descriptions and not just placeholder stuff.

File: stackpulse/create-incident-war-room-pagerduty/playbook.yaml [hunk 4]
```diff
@@ -10,11 +10,42 @@ parameters:
     type: var
     description: "The emails of the users that will be invited to the channel (by default, only the incident reporter)."
     default: '{{ $.event.Reporter }}'
+  - name: reporter
```
Comment (hagaishapira) [inline stackpulse/create-incident-war-room-pagerduty/playbook.yaml:13]: These descriptions are not verbose and helpful enough, please think of this like someone who is just using this playbook out of the box trying to create a war room for his incident

File: stackpulse/create-incident-war-room-pagerduty/playbook.yaml [hunk 5]
```diff
@@ -10,11 +10,42 @@ parameters:
...
+    default: '{{ $.event.Severity }}'  
+  - name: warRoomChannelName
+    type: var
+    description: The War Room Slack channel name to be attached to the incident.
   - name: zoomIntegrationName
     type: integration
     description: Zoom integration.
     options:
       integration_type: zoom
+  - name: slackOnCaller
```
Comment (hagaishapira) [inline stackpulse/create-incident-war-room-pagerduty/playbook.yaml:41]: I don't think we need this parameter, the use for this is to ask the *reporter* if the page the pagerduty on caller.

**Review by hagaishapira — COMMENTED** [review]
Please review the parameter descriptions, these really matter as they are what explains to new user what is this parameter and what they should input there.

File: stackpulse/create-incident-war-room-opsgenie/playbook.yaml (hunk 2 above)
Comment (moshebe) [reply stackpulse/create-incident-war-room-opsgenie/playbook.yaml:11]: I personally prefer no quotes I think it looks cleaner.
But there are cases when we use special characters so we need to wrap it with quotes.
Do you prefer all the textual fields will be wrapped in this case?

**Comment by hagaishapira** [comment]
I think that in a playbook we are kinda forced to use quotes and that the
backend adds them either way. I know it is prettier without but we need
more consistency.

**Review by hagaishapira — APPROVED** [review]
Very very nice!

**Pull request merged**